nixos/modules/virtualisation/virtualbox-guest.nix

```diff
@@ -32,6 +32,12 @@ let
 in
 {
   ###### interface
+  imports = [
+    (mkRenamedOptionModule
+      [ "virtualisation" "virtualbox" "guest" "draganddrop" ]
+      [ "virtualisation" "virtualbox" "guest" "dragAndDrop" ]
+    )
```

Comment:
```suggestion
    (mkRenamedOptionModule [ "virtualisation" "virtualbox" "guest" "draganddrop" ] [ "virtualisation" "virtualbox" "guest" "dragAndDrop" ])
```

Reply:
When you have a few of those, it is always easier to write them into one line

Reply:
A matter of taste I suppose.

Reply:
probably :)